cmd/common_konnect.go

```diff
@@ -129,6 +129,7 @@ func resetKonnectV2(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	dumpConfig.IsConsumerGroupScopedPluginSupported = true
```

Comment:
Why is this true only in `resetKonnectV2` function while in the others (e.g. `dumpKonnectV2`) it's not? How does it correspond with the fact that Konnect support for CG-scoped plugins was not released yet?

Reply:
Actually we don't need this flag set here. We only need when reading from a configuration file (so when running `sync`) as opposite of `reset` and `dump`. I'll remove it.